### PR TITLE
Rename DOCKER_CONTAINER_IMAGE => DOCKER_NOTEBOOK_IMAGE

### PR DESCRIPTION
**Pull request by jtyberg**
Renamed the environment variable used to specify the image from which to spawn Notebook server containers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -215,38 +215,37 @@ Use `docker logs <container>`.  For example, to view the logs of the `jupyterhub
...
 Notebook servers. You set the desired Notebook server image in a
-`DOCKER_CONTAINER_IMAGE` environment variable.
+`DOCKER_NOTEBOOK_IMAGE` environment variable.
 
-When spawning single-user notebooks, JupyterHub reads the DockerSpawner's 
-image name from `jupyterhub_config.py` file. This file includes the
-DockerSpawner's image name which refers to desired image in the
-`DOCKER_CONTAINER_IMAGE` environment variable:
+JupyterHub reads the Notebook image name from `jupyterhub_config.py`, which
+reads the Notebook image name from the `DOCKER_CONTAINER_IMAGE` environment
```
Comment (willingc) [inline README.md]: Shouldn't this be ``DOCKER_NOTEBOOK_IMAGE``?

**Comment by willingc** [comment]
@jtyberg Thanks. Looks good to me (just one question if we missed one change).

**Comment by jtyberg** [comment]
Doh!  So much for my search skills.  Fixed.

**Comment by willingc** [comment]
Thanks @jtyberg! 😎

**Pull request merged**